notebooks/279-photo-maker/279-photo-maker.ipynb

```diff
@@ -0,0 +1,1072 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    transformers diffusers gradio openvino-nightly "git+https://github.com/huggingface/optimum-intel.git"
</pre><p>I do not see optimum usage in this notebook?</p><p>is it really need?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1632/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
removed